app/routes/__frontend/resources/virtual-coffee-handbook/guides-to-virtual-coffee/glossary.mdx

```diff
@@ -0,0 +1,110 @@
+---
```

Comment:
Would love to see a link to this one on the home page under All Things Virtual Coffee or on the About page.

Reply:
I still need to figure out where the best to put it 🤔

Reply:
🤔 yeah, could be a future update.